src/elements/content-sidebar/activity-feed/version/__tests__/Version.test.js

```diff
@@ -2,10 +2,11 @@ import * as React from 'react';
 import { shallow } from 'enzyme';
 import selectors from '../../../../common/selectors/version';
 import { VersionBase as Version } from '../Version';
-import { PLACEHOLDER_USER, VERSION_UPLOAD_ACTION } from '../../../../../constants';
+import { FILE_REQUEST_NAME, PLACEHOLDER_USER, VERSION_UPLOAD_ACTION } from '../../../../../constants';
+import messages from '../../../../common/messages';
 
 const translationProps = {
-    intl: { formatMessage: () => {} },
+    intl: { formatMessage: anyString => anyString },
```

Comment:
Where is `anyString` defined?

Reply:
it's just a function argument that makes formatMessage return whatever is passed in